docs/GUIDE.md

```diff
@@ -99,6 +100,18 @@
 
 You generally want to adhere to the same extensions and GHC options the [Plutarch repo](https://github.com/Plutonomicon/plutarch/blob/master/plutarch.cabal) uses.
 
+### Using the Plutarch Prelude
```

Comment:
@L-as can we merge `master` to `staging` to get the new guide structure but keep the tests intact?

Reply:
we can't, this needs some git magickery which I'll do now.